extensions/monday/package.json

```diff
@@ -0,0 +1,67 @@
+{
+  "$schema": "https://www.raycast.com/schemas/extension.json",
+  "name": "monday.com",
+  "title": "monday.com",
+  "description": "Quickly navigate to your latest monday.com boards and create items",
+  "icon": "command-icon.png",
+  "author": "freak4pc",
+  "license": "MIT",
+  "commands": [
+    {
+      "name": "index",
+      "title": "Recent Boards",
+      "subtitle": "monday.com",
```

Comment:
```suggestion
      "subtitle": "monday.",
```

Reply:
The brand name is monday.com (you can see the website), prefer leaving it as-is :)